src/odometry/odometry/odometry.py

```diff
@@ -23,6 +17,11 @@ def main():
 
 
 class Odometry(Node):
+    # --------------------------------
+    # -- robot kinematic parameters --
+    # --------------------------------
+    WHEEL_RADIUS = 9.8 * 0.01 / 2 # m
+    WHEEL_SEPARATION = 31 * 0.01 # m
```

Comment:
How did you compute WHEEL_RADIUS and WHEEL_SEPARATION?

Reply:
The measured diameter was 9.8cm